bindings/c/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ if(BUILD_TESTING)
     # Add unit test
     add_executable(cparameterUnitTest Test.cpp)
 
-    target_link_libraries(cparameterUnitTest PRIVATE catch cparameter pfw_utility)
+    target_link_libraries(cparameterUnitTest PRIVATE cparameter pfw-test-toolbox)
```

Comment:
pfw_utility => test-utility ? (why is it prefixed by `pfw-` anyway (it is not a public lib))